.github/workflows/cleanup-terraform.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: cleanup-terraform
```

Comment:
Similar to [our dbt setup](https://github.com/ccao-data/data-architecture/blob/a5684e69b6534c02a9e13a01497ed9e3236d0d46/.github/workflows/cleanup_dbt_resources.yaml#L1), this workflow destroys all staging resources when a PR is closed. For an example of a successful run of this workflow, [see here](https://github.com/ccao-data/model-sales-val/actions/runs/6386009283/job/17332052885).